ci/scale-test.sh

```diff
@@ -30,6 +30,7 @@ $OVNSUDO docker exec ovn-rally rally task report $TASKID --out /root/create-netw
 $OVNSUDO docker exec ovn-rally rally task results $TASKID > ./create-networks-data.json
 $OVNSUDO docker cp ovn-rally:/root/create-networks-output.html .
 $OVNSUDO docker exec ovn-rally rally task delete --uuid $TASKID
+$OVNSUDO rm -rf /tmp/rally-ovs-output.raw
```

Comment:
No, they are not. But I want to verify the results from last run are cleared so that the check_ovn_rally is not confused. Thanks.